extension-manifest-v3/test/store/setup.js

```diff
@@ -0,0 +1,4 @@
+import './pre-setup.js';
```

Comment:
pre-setup is needed so chrome is present as a global before `webextension-polyfill` is loaded.